### DISCUSSION TITLE
why is the rendering quality so bad?

### DISCUSSION THREAD

**Discussion by PatrickDDj** [lsongx/nerfplayer-nerfstudio#13] [Q&A] [answered]
<img width="866" alt="Screen Shot 2023-06-17 at 19 25 54" src="https://github.com/lsongx/nerfplayer-nerfstudio/assets/55349276/acfe5acc-18b3-4d4d-96b8-36d226295a3c">
I use nerfacto backbone, while getting foggy images and videos. 
Does nerfplayer need other setting in nerfstudio?

**Comment by lsongx** [comment, marked as answer]
Maybe you could try other methods like dnerf or kplanes, and see if their quality meets your expectations? I don’t have comments on improving nerfplayer for now.

**Comment by orrblue** [comment]
I'm pretty sure to see the true quality you need to be in Render View in Nerf Studio. That requires adding a camera, setting its resolution (default is 1080p I think), and then clicking on Render View (rather than Viewport). The Viewport mode is probably showing lower quality to allow interactive navigation within the scene. Does that help?

**Comment by xiazhi1** [comment]
> <img alt="Screen Shot 2023-06-17 at 19 25 54" width="866" src="https://user-images.githubusercontent.com/55349276/246604604-acfe5acc-18b3-4d4d-96b8-36d226295a3c.png"> I use nerfacto backbone, while getting foggy images and videos. Does nerfplayer need other setting in nerfstudio?

Hi, May I bother you to share your environment for configuring nerfplayer (cuda, torch, nerfstudio), I am a fresher in the NeRF field,Looking forword to your reply!

**Comment by xiazhi1** [comment]
@PatrickDDj

**Comment by xiazhi1** [comment]
> > <img alt="Screen Shot 2023-06-17 at 19 25 54" width="866" src="https://user-images.githubusercontent.com/55349276/246604604-acfe5acc-18b3-4d4d-96b8-36d226295a3c.png"> I use nerfacto backbone, while getting foggy images and videos. Does nerfplayer need other setting in nerfstudio?
> 
> Hi, May I bother you to share your environment for configuring nerfplayer (cuda, torch, nerfstudio), I am a fresher in the NeRF field,Looking forword to your reply!

My bug is just like this issue https://github.com/nerfstudio-project/nerfstudio/issues/2226#issue-1804747144

**Comment by xiazhi1** [comment]
> 

The bug is solved by https://github.com/nerfstudio-project/nerfstudio/issues/2226#issuecomment-1646393393
